### PR TITLE
Fixes a few googles not working

### PR DESCRIPTION
**Pull request by ShiftyRail**
probably doesn't fix the issue #30640 but does fix mesons, night visions, and thermals.
Didn't test MoMMis.

Issue was that Deity changed the way alphas worked.

### CODE REVIEW

**Comment by ghost** [comment]
the goggles
they no longer do nothing

File: code/datums/gamemode/role/vampire_role.dm [hunk 1]
```diff
@@ -300,6 +300,9 @@
 			O.status &= ~ORGAN_BLEEDING
 	nullified = max(0, nullified - 1)
 
+	if (locate(/datum/power/vampire/mature) in current_powers)
+		H.dark_plane.alphas["vampire_vision"] = 255
```
Comment (DeityLink) [inline code/datums/gamemode/role/vampire_role.dm:305]: that's gonna be reset by human/update_perception()

A good fix would be to have a role/update_perception(), fit that in the vampire subtype, and call the roles' update_perception() in human/update_perception() after the reset.

Comment (ShiftyRail) [reply code/datums/gamemode/role/vampire_role.dm:305]: Vampires get that again updated on their `Life()` proc but yeah

**Comment by ShiftyRail** [comment]
All done, can be merged

**Pull request merged**